src/Fields/FileField.php

```diff
@@ -31,8 +31,10 @@ public function __construct(array $args = array())
 
     public function validate($value)
     {
-        if (0 == $value->size && !$this->required) {
+        if ((!isset($value->size) && $this->isDisabled()) || 0 == $value->size && !$this->required) {
```

Comment:
Remove isDisabled and combine the condition as:
```php
if ((!isset($value->size) || 0 == $value->size) && !$this->required
```

Reply:
Changed.